__tests__/login/oidc/oidcHandlers/AuthorizationCodeWithPkceOidcHandler.spec.ts

```diff
@@ -89,5 +89,30 @@ describe("AuthorizationCodeWithPkceOidcHandler", () => {
       );
       expect(session.neededAction).toMatchObject(RedirectorResponse);
     });
+
+    it("Handles login when a client secret is present", async () => {
+      const authorizationCodeWithPkceOidcHandler = getAuthorizationCodeWithPkceOidcHandler();
+      const oidcOptions: IOidcOptions = {
+        ...standardOidcOptions,
+        client: {
+          ...standardOidcOptions.client,
+          clientSecret: "I'm can't cook because I only drink Soylent"
```

Comment:
```suggestion
          clientSecret: "I can't cook because I only drink Soylent"
```
Client secrets: a developer's diary.